console/CleanClusterKeysFile.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Initbiz\CumulusCore\Console;
+
+use Config;
+use Storage;
+use Illuminate\Console\Command;
+use Initbiz\CumulusCore\Models\Cluster;
+
+class CleanClusterKeysFile extends Command
```

Comment:
Is it not also depreciated?

Reply:
You mean `Illuminate\Console\Command` or what?